manifests/0000_25_kube-scheduler-operator_07_clusteroperator.yaml

```diff
@@ -36,3 +36,6 @@ status:
     - group: controlplane.operator.openshift.io
       resource: podnetworkconnectivitychecks
       namespace: openshift-kube-scheduler
+    - group: "rbac.authorization.k8s.io"
+      name: "system:openshift:operator:cluster-kube-scheduler-operator"
+      resource: "clusterrolebindings"
```

Comment:
The CVO will only apply these if it sees this manifest when the ClusterOperator doesn't exist.  That means these won't get injected into existing clusters updating into this new code.  Do you not reconcile these as part of your Go controller's ClusterOperator `status` management?  Or does this manifest somehow also feed your Go controller?  Or...?

Reply:
You are right, I will mark this as WIP and research it a bit more.